schedule/ha/bv/basic_cluster_node.yaml

```diff
@@ -97,3 +98,17 @@ conditional_schedule:
     HA_GRACEFUL_SHUTDOWN:
       1:
         - ha/graceful_shutdown
+  create_hdd_tests:
+    CREATE_HDD:
+      1:
+        - migration/deregister_system
+        - ha/prepare_shutdown
+        - console/force_scheduled_tasks
+        - shutdown/grub_set_bootargs
+        - shutdown/cleanup_before_shutdown
+        - shutdown/shutdown
+        - '{{svirt_upload_assets}}'
+  svirt_upload_assets:
```

Comment:
Sorry, I just noticed that, where is this condition `svirt_upload_assets` used?

Reply:
Only for s390x (when BACKEND=svirt). I added that one there as it was also present in the create_hdd schedule, but I don't think we ever reach this condition with this schedule. When we work on generating qcwo2 images for s390x we'll know for sure and can perhaps drop these lines.